doc/source/whatsnew/v0.20.2.txt

```diff
@@ -37,6 +37,7 @@ Bug Fixes
 ~~~~~~~~~
 
 - Bug in using ``pathlib.Path`` or ``py.path.local`` objects with io functions (:issue:`16291`)
+- Bug in using ``DataFrame.update`` when overwrite=False and the second value is nan (:issue:`15593`)
```

Comment:
add the 2nd issue as well. 

Bug in ``DataFrame.update()`` with ``overwrite=False`` and ``NaN`` values